engine/admin/index.md

```diff
@@ -75,6 +75,133 @@ restart Docker. This method works for every Docker platform. The following
 }
 ```
 
+## Isolate containers within a user namespace
```

Comment:
Wondering if;

- this topic should be in the "security" section
- it should be on its own page

If there's a dedicated page for this, we can copy some more of the technical information from https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-user-namespace-options to the page, and remove it from the `dockerd` reference.

The reference docs contain quite some information that's important when using this feature (limitations, how to start a privileged container if userns is enabled, and the current limitation of having a single "global" mapping, instead of custom mappings per container)

If we migrate the content here, we can keep a short section in the daemon docs, but refer to the extended documentation elsewhere